LICENSE-BSD-3-Clause

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Nydus Authors. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+//    * Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//    * Redistributions in binary form must reproduce the above
+// copyright notice, this list of conditions and the following disclaimer
+// in the documentation and/or other materials provided with the
+// distribution.
+//    * Neither the name of Google Inc. nor the names of its
```

Comment:
Google Inc.?

https://opensource.org/licenses/BSD-3-Clause

Reply:
Good catch, thanks! That one was copied from rust-vmm :(

Reply:
That's where we need to tweak the BSD-3-Clause license:(